crates/server-primitives/src/admin.rs

```diff
@@ -1,13 +1,34 @@
+use calimero_context::ContextManager;
 use calimero_primitives::application::{Application, ApplicationId};
 use calimero_primitives::context::{Context, ContextId};
 use calimero_primitives::hash::Hash;
 use calimero_primitives::identity::{PublicKey, WalletType};
+use calimero_store::Store;
 use camino::Utf8PathBuf;
+use libp2p::identity::Keypair;
 use semver::Version;
 use serde::{Deserialize, Serialize};
 use serde_json::Value;
 use url::Url;
 
+#[derive(Debug)]
+#[non_exhaustive]
+pub struct AdminState {
+    pub store: Store,
+    pub keypair: Keypair,
+    pub ctx_manager: ContextManager,
+}
```

Comment:
This will only be used by `calimero-server`, it should be there instead.

Primitive crates are supposed to host structures/behaviour that aid interaction across crates.